lib/page/assets/signal/signalDetailPage.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:polka_wallet/common/components/goPageBtn.dart';
+import 'package:polka_wallet/common/components/subTitle.dart';
+import 'package:polka_wallet/page/assets/signal/signalResultPage.dart';
+import 'package:polka_wallet/store/app.dart';
+import 'package:polka_wallet/utils/i18n/index.dart';
+
+class SignalDetailPage extends StatefulWidget {
+  SignalDetailPage(this.store);
+
+  static final String route = '/assets/signal/detail';
+  final AppStore store;
+
+  @override
+  _DetailPageState createState() => _DetailPageState(store);
+}
+
+class _DetailPageState extends State<SignalDetailPage> {
+  _DetailPageState(this.store);
+
+  final AppStore store;
+
+  @override
+  Widget build(BuildContext context) {
+    var dic = I18n.of(context).assets;
+    
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(dic['signal.tokens'])
+      ),
+      body: SafeArea(
+        child: Builder(
+          builder: (BuildContext context) {
+          return ListView(
+            shrinkWrap: true, 
+            padding: const EdgeInsets.all(20),
+            children: <Widget>[
+              Text(
+                dic['transaction.message'],
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 10),
+                child: Text(
+                  dic['signal.instruction'],
+                  textAlign: TextAlign.center,
+                ),
+              ),
+              subTitle(dic['formula']),
+              Row(
+                children: <Widget>[
+                  formulaInput(
+                    lable: dic['lock.duration']
+                  ),
+                  Text(' , '),
+                  Expanded(
+                    child: Column(
+                      children: <Widget>[
+                        TextField(
+                          textAlign: TextAlign.center,
+                        ),
+                        Container(
+                          padding: const EdgeInsets.symmetric(vertical: 5),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: <Widget>[
+                            Text(
+                              '${dic["signal"]},${dic["lock"]}',
+                              style: TextStyle(
+                                fontSize: 10
+                              )
+                            ),
+                            Icon(Icons.arrow_drop_down)
+                          ]),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Text(' , '),
+                  formulaInput(
+                    lable: dic['amount.tokens']
+                  ),
```

Comment:
On this 1st "Signal Tokens" page, please add an additional input field at the end of the formula with label "DHX public key (hex)" and provide a link below it with text "Guide on how to create a public key".